tests/integration_tests/scheduler/bin/bjobs.py

```diff
@@ -31,6 +31,7 @@ def get_parser() -> argparse.ArgumentParser:
         description="Mocked LSF bjobs command reading state from filesystem"
     )
     parser.add_argument("jobs", type=str, nargs="*")
+    parser.add_argument("-w", action="store_true")
```

Comment:
Belongs to a different commit?

Reply:
No, it failed because of test_lsf_driver.py::test_job_name. When running towards the real cluster; it would shorten the job name, so I added `-w` to bjobs in the test. This then caused the mocked command to fail due to unknown flags.